src/sinks/aws_cloudwatch_metrics/mod.rs

```diff
@@ -309,13 +309,13 @@ impl Service<PartitionInnerBuffer<Vec<Metric>, String>> for CloudWatchMetricsSvc
     type Future = BoxFuture<'static, Result<Self::Response, Self::Error>>;
 
     fn poll_ready(&mut self, _cx: &mut Context) -> Poll<Result<(), Self::Error>> {
-        // Emission of Error internal event is handled upstream by the caller
+        // Emission of an internal event in case of errors is handled upstream by the caller.
```

Comment:
Not opposed to changing the wording as long as we are consistent - I think we've added this same comment to a few other sinks since this PR was initially submitted, The gcs, elasticsearch, azure common. Worth grepping for.